docker/mbzirc_sim/Dockerfile

```diff
@@ -115,14 +130,18 @@ ENV IGNITION_VERSION fortress
 
 RUN sudo apt-get update \
   && rosdep update \
-  && rosdep install -r --from-paths src -i -y --rosdistro galactic
+  && rosdep install -r --from-paths src -i -y --rosdistro galactic \
+  && sudo rm -rf /var/lib/apt/lists/* \
+  && sudo apt-get clean -qq
 
-RUN /bin/bash -c 'source /opt/ros/galactic/setup.bash && colcon build --merge-install'
+RUN /bin/bash -c 'source /opt/ros/galactic/setup.bash \
+  && colcon build --merge-install' \
+  && rm -fr build/ log/
 RUN /bin/sh -c 'echo ". /opt/ros/galactic/setup.bash" >> ~/.bashrc' \
  && /bin/sh -c 'echo ". ~/mbzirc_ws/install/setup.sh" >> ~/.bashrc'
 
-# Create a directory required by cloudsim
-RUN mkdir -p /tmp/ign
+# Leave apt-get ready for users
```

Comment:
Not sure how would be the interaction of the users with this docker software. 
 * Do they need to install anything? If not we can remove this.
 * Should the run script be run automatically when users run the docker layer? if so, we need enable lines below.

Reply:
ack, afeb04c .